docs/source/user_guide.rst

```diff
@@ -0,0 +1,184 @@
+User Guide
+==========
+
+Imaging inverse problems are described by the equation :math:`y = \noise{\forw{x}}` where
+:math:`x` is an unknown image we want to recover,
+:math:`y` are the observed measurements, :math:`A` is a
+deterministic (linear or non-linear) operator capturing the physics of the acquisition and
+:math:`N` characterizes the noise affecting the measurements.
+
+Operators
+~~~~~~~~~
+
+The library provides a large variety of imaging operators ``physics`` modelling :math:`\noise{\forw{\cdot}}`,
+which can simulate the observation process:
+
+::
+
+    x = load_image()
+    y = physics(x) # simulate observation
+
+.. grid:: 3
+    :gutter: 1
+
+    .. grid-item-card::
+        :link: physics
+        :link-type: ref
+
+        :octicon:`device-camera-video` **Physics**
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Define forward operators and noise models.
+
+Reconstruction
+~~~~~~~~~~~~~~
+
+In order to recover an image from its measurements, the library provides many
+reconstruction methods :math:`\hat{x}=R(y, A)`, which often leverage knowledge of the acquisition ``physics``:
+
+::
+
+    x_hat = model(y, physics)
+
+
+.. grid:: 3
+    :gutter: 1
+
+    .. grid-item-card::
+        :link: denoisers
+        :link-type: ref
+
+        :octicon:`package` **Denoisers**
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Classical and deep denoisers with
+        pretrained weights.
+
+    .. grid-item-card::
+        :link: artifact
+        :link-type: ref
+
+        :octicon:`law` **Artifact Removal**
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Learned filtered back-projection.
+
+    .. grid-item-card::
+        :link: optim
+        :link-type: ref
+
+        :octicon:`rocket` **Optimization**
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Priors and data-fidelity functions,
+        and optimization algorithms.
+
+    .. grid-item-card::
+        :link: unfolded
+        :link-type: ref
+
+        :octicon:`diff-renamed` **Unfolded Algorithms**
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Unfolded architectures.
+
+    .. grid-item-card::
+        :link: iterative
+        :link-type: ref
+
+        :octicon:`play` **Iterative Reconstruction**
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Plug-and-play, RED, and deep image prior.
+
+
+    .. grid-item-card::
+        :link: sampling
+        :link-type: ref
+
+        :octicon:`flame` **Sampling**
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Diffusion and Langevin algorithms.
+
+Training, Testing and Utilities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reconstruction models :math:`R` can be trained on datasets to improve their performance:
```

Comment:
```suggestion
Reconstruction models ``model`` can be trained on datasets to improve their performance:
```
Not sure if we want to keep the :math:`R` notation here since we are not using it afterward.